addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -1,16 +1,11 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
 from collections import defaultdict
-from urllib3.util.ssl_ import create_urllib3_context
-from urllib3.contrib.pyopenssl import inject_into_urllib3
-from OpenSSL.crypto import load_certificate, load_privatekey, FILETYPE_PEM
 
 from odoo import fields, models, _
 from odoo.exceptions import UserError
 from odoo.tools import html_escape, zeep
 from odoo.tools.float_utils import float_round
+from odoo.addons.certificate.tools import CertificateAdapter
```

Comment:
Like before, don't mind this if you don't need end up pushing again

```suggestion

from odoo.addons.certificate.tools import CertificateAdapter
```

(And if you really want to push it, the three imports after this one need to be the very first and the `_`import comes before `fields` :face_in_clouds:)